OpenRA.Mods.RA2/Traits/ChangeOwnerOnGarrisoner.cs

```diff
@@ -0,0 +1,60 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2019 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.RA2.Traits
+{
+    public class ChangeOwnerOnGarrisonerInfo : ChangeOwnerInfo, ITraitInfo, Requires<GarrisonableInfo>
+    {
+        [Desc("The speech notification on enter first garrisoner on garrisonable.")]
+        public readonly string EnterNotification = null;
+
+        [Desc("The speech notification on exit last garrisoner on garrisonable")]
```

Comment:
Fullstop missing.

I'd also go with something like
`Speech notification played when the first actors enters/leaves this garrison.`

Reply:
I tried looking at ITraitInfo to see more information on where this is implemented / referenced. and I can't seem to find anything. I'm guessing this is something DnApp put in as a place holder.

Reply:
L48 and L56 uses these within this very own file.